packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -70,6 +70,8 @@ export namespace TerminalMenus {
     export const TERMINAL_TASKS_CONFIG = [...TERMINAL_TASKS, '4_terminal'];
     export const TERMINAL_NAVIGATOR_CONTEXT_MENU = ['navigator-context-menu', 'navigation'];
     export const TERMINAL_OPEN_EDITORS_CONTEXT_MENU = ['open-editors-context-menu', 'navigation'];
+
+    export const TERMINAL_CONTEXT_MENU = ['terminal'];
```

Comment:
minor: we should add `-context-menu` so it is a little clearer and has less change for collisions:

```suggestion
    export const TERMINAL_CONTEXT_MENU = ['terminal-context-menu'];
```